includes/block-options/isc-image-block-edit-link.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Add a link to the media modal when Image Source Control fields for image blocks are disabled.
+ *
+ * @param {object} wp - The WordPress global object.
+ */
+
+( function ( wp ) {
+	'use strict';
+
+	var __                          = wp.i18n.__;
+	var el                          = wp.element.createElement;
+	var createHigherOrderComponent  = wp.compose.createHigherOrderComponent;
+	var MediaUpload                 = wp.blockEditor.MediaUpload;
+	var PanelBody                   = wp.components.PanelBody;
+	var InspectorControls           = wp.blockEditor.InspectorControls;
```

Comment:
**Suggestion:** You can use destructuring assignment to simplify the access to the `wp` object properties. [best practice]
```suggestion
 const { __, createElement: el, createHigherOrderComponent, MediaUpload, PanelBody, InspectorControls } = wp;
```